process/css-styleguide/heading/heading.css

```diff
@@ -0,0 +1,186 @@
+@import "../default.css";
+
+.inter {
+  font-family: "Inter", sans-serif;
+}
+
+.poppins {

```

Comment:
As I said in the previous comment, "Poppins should be used for headings and Inter for the rest.". You could maybe get rid of the `.poppins` class and add directly the font-family to the `.typography` class. 

Also, there is no need of an `.inter` class in this file